### DISCUSSION TITLE
ender 7 open source document?

### DISCUSSION THREAD

**Discussion by sgassi** [community/community#40776] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
I have an Ender7 and I am totally happy.
But now I would like to optimize it in a few places and should have the schematic.
I would like to drive full speed but with the original hardware it is not possible. 
Please make the schematic open-source on your GitHub. Thanks BR Steffen

**Comment by ossanna16** [comment]
Hi there @sgassi and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
